core/src/main/java/org/apache/accumulo/core/metadata/schema/RootTabletMetadata.java

```diff
@@ -97,15 +97,6 @@ private Data(int version, TreeMap<String,TreeMap<String,String>> columnValues) {
       this.columnValues = columnValues;
     }
 
-    public int getVersion() {
-      return version;
-    }
-
-    public static boolean needsUpgrade(final String json) {
-      var rootData = GSON.get().fromJson(json, Data.class);
-      int currVersion = rootData.getVersion();
-      return currVersion < VERSION;
-    }
```

Comment:
Do we read the version from the serialized metadata elsewhere now? It seems important.